src/handlers.ts

```diff
@@ -14,7 +14,7 @@ import {
 } from './utils';
 
 const { isArray } = Array;
-const slice = Function.prototype.bind.call(Function.prototype.bind, Array.prototype.slice);
+const slice = Function.prototype.bind.call(Function.prototype.call, Array.prototype.slice);
```

Comment:
This is the fix, the first param to this `bind.call` should be `Function.prototype.call`.